vnext/ReactUWP/Views/ShadowNodeBase.cpp

```diff
@@ -112,4 +130,74 @@ void ShadowNodeBase::UpdateTransformPS()
   }
 }
 
+void ShadowNodeBase::UpdateHandledKeyboardEvents(KeyboardType keyboardType, folly::dynamic const& value)
+{
+  if (keyboardType == KeyboardType::KeyDown) {
+    m_handledKeyDownKeyboardEvents = KeyboardHelper::FromJS(value);
+  }
+  else
+  {
+    m_handledKeyUpKeyboardEvents = KeyboardHelper::FromJS(value);
+  }
+
+  UpdateKeyboardEventHooks(m_view);
+}
+
+void ShadowNodeBase::UpdateKeyboardEventHooks(XamlView const& xamlView)
```

Comment:
>XamlView const& xamlView [](start = 46, length = 24)

This parameter isn't used.  Did you mean to use it instead of m_view below?

Reply:
It's used to serve UpdateKeyboardEventHooks(null) and UpdateKeyboardEventHooks(m_xamlView).
new implementation doesn't need this any more.